contrib/perf/generate_perf_report.py

```
@@ -0,0 +1,377 @@
+        if not start_time or not end_time:
+            return "-"
+        
+        try:
+            start = datetime.fromisoformat(start_time.replace('Z', '+00:00'))
+            end = datetime.fromisoformat(end_time.replace('Z', '+00:00'))
+            
+            if start.date() == end.date():
+                return f"{start.strftime('%m/%d/%Y')}"
+            else:
+                return f"{start.strftime('%m/%d')} - {end.strftime('%m/%d')}"
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)